features/FEATURE_BLE/ble/pal/MemorySecurityDb.h

```diff
@@ -276,8 +276,16 @@ class MemorySecurityDb : public SecurityDb {
         for (size_t i = 0; i < MAX_ENTRIES; i++) {
             if (_entries[i].state == ENTRY_FREE) {
                 continue;
-            } else if (peer_address == _entries[i].peer_identity.identity_address
-                && _entries[i].flags.peer_address_is_public == peer_address_public) {
+            } else if (
```

Comment:
looks like you already fixed this in a different PR?

Reply:
Not sure about that; we talked about it for sure. If it was fixed then why is there a diff and a new condition ?